src/ontology/mondo.Makefile

```diff
@@ -1310,6 +1310,72 @@ config/exclusion_reasons.tsv:
 
 all: config/exclusion_reasons.tsv
 
+# ----------------------------------------
+# Babelon Translation Files
+# ----------------------------------------
+
+BABELONPY=babelon -q
+TRANSLATIONS_OWL=$(TRANSLATIONSDIR)/mondo-jp.babelon.owl
+TRANSLATIONS_TSV=$(TRANSLATIONSDIR)/mondo-jp-preprocessed.babelon.tsv
+
+translations:
+	$(MAKE) $(TRANSLATION_FILES)
```

Comment:
@matentzn in Slack you mentioned this was folded into the release pipeline. Where is that done?

Reply:
Answered [above](https://github.com/monarch-initiative/mondo/pull/7928/files#r2015006091) - it's part of `MAIN_PRODUCTS`